src/main/java/network/brightspots/rcv/Logger.java

```diff
@@ -99,9 +99,11 @@ static void setup() {
 
   // adds file logging for a tabulation run
   static void addTabulationFileLogging(String outputPath) throws IOException {
-    // use Level.FINE to capture audit info
-    tabulationHandler =
-        new FileHandler(outputPath, LOG_FILE_MAX_SIZE_BYTES, TABULATION_LOG_FILE_COUNT, true);
+    // replace any instances of % with %% because the FileHandler pattern generator requires this
+    tabulationHandler = new FileHandler(outputPath.replace("%", "%%"),
```

Comment:
Would this work as intended if a file name has a double percentage in the name?

Reply:
Yep, I tried this with multiple percent characters and that part works, but as I'm checking again I realize I broke the %g functionality here so it's not quite correct yet.  Will push a fix for that.

Reply:
Just pushed a fix for this bit.  Go ahead and get hog-wild with your path names.